pkg/agent/client.go

```diff
@@ -250,11 +253,35 @@ func (a *Client) Connect() (int, error) {
 	}
 	a.conn = conn
 	a.stream = stream
+	a.serverChannel = make(chan *client.Packet, xfrChannelSize)
+	a.serverError = nil
 	a.serverID = serverID
+	go a.writeToProxyServer()
 	klog.V(2).InfoS("Connect to", "server", serverID)
 	return serverCount, nil
 }
 
+func (a *Client) writeToProxyServer() {
+	defer func() {
+		if panicInfo := recover(); panicInfo != nil {
+			klog.V(2).InfoS("Exiting writeToProxyServer with recovery", "panicInfo", panicInfo, "serverID", a.serverID)
+		} else {
+			klog.V(2).InfoS("Exiting writeToProxyServer", "serverID", a.serverID)
```

Comment:
v=2 seems a little low for this doesn't it? I would expect this to be v=4 at least